src/send/newadd.js

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import Button from '@mui/material/Button'
+import TextField from '@mui/material/TextField'
+import { useRef } from 'react'
+import Stack from '@mui/material/Stack'
+
+import SendIcon from '@mui/icons-material/Send'
+
+function NewAdd() {
+	const titleInputRef = useRef()
+	const descriptionInputRef = useRef()
+
+	function goTo(event) {
+		event.preventDefault.useState()
+
+		const enteredTitle = titleInputRef.current.value
+		const enteredDescription = descriptionInputRef.current.value
+
+		const goData = { title: enteredTitle, description: enteredDescription }
+
+		console.log(goData)
+	}
```

Comment:
rewrite to `handleSubmit` function. 

Function should be invoked after pressing send button. 

temporarily console.log defined state, which should be changed in textfield